CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -516,12 +516,10 @@ public function buildQuickForm() {
       $taxTerm = CRM_Utils_Array::value('tax_term', $invoiceSettings);
       foreach ($this->_lineItem as $key => $value) {
         foreach ($value as $k => $v) {
-          if (isset($v['tax_rate'])) {
-            if ($v['tax_rate'] != '') {
-              $getTaxDetails = TRUE;
-              // Cast to float to display without trailing zero decimals
-              $tplLineItems[$key][$k]['tax_rate'] = (float) $v['tax_rate'];
-            }
+          if (!empty($v['tax_rate'])) {
```

Comment:
This is just a cleanup I think? check for empty in place of check for isset + !== ''